xchange-okcoin/src/main/java/com/xeiam/xchange/okcoin/service/streaming/OkCoinBaseStreamingService.java

```diff
@@ -68,4 +66,16 @@ public void send(String msg) {
   public READYSTATE getWebSocketStatus() {
     return READYSTATE.OPEN;
   }
+
+  public WebSocketBase getSocketBase() {
+    return socketBase;
+  }
+
+  public void setSocketBase(WebSocketBase socketBase) {
```

Comment:
This method should only be used for dependency injection during testing I suppose. Then it should be made pakcage private, not public. Alternatively, I would consider creation of package private constructor to be used in tests